moderation/assets/moderation.html

```diff
@@ -472,6 +472,8 @@ <h4>Delete ALL server warnings?</h4>
 
         {{checkbox "WarnIncludeChannelLogs" "WarnIncludeChannelLogs" "Create message logs in the channel that the command was run in when a user is warned" .ModConfig.WarnIncludeChannelLogs}}
         {{checkbox "WarnSendToModlog" "WarnSendToModlog" "Send warnings to the modlog" .ModConfig.WarnSendToModlog}}
+        {{checkbox "UnwarnSendToModlog" "UnwarnSendToModlog" "Send warning removal to the modlog" .ModConfig.UnwarnSendToModlog}}
```

Comment:
Have you considered a horizontal rule here?

Reply:
Nop, but that's a good idea

Reply:
After reviewing the page, I don't think that's a good idea, there is already two columns on the page

Reply:
A horizontal rule would just separate vertically, you can see there’s one just a few lines below this. It’s just a line with a little bit of spacing that goes across the column, not creating a new column.

Reply:
I misread, yeah I'll add that